spec/javascripts/javascript_spec.js

```diff
@@ -0,0 +1,55 @@
+
+describe("Canarytest", function(){
+  it("should work", function(){
+    expect(true).toBe(true);
```

Comment:
Please remove this test after ensuring that Jasmine functions properly.  True == True always